package.json

```diff
@@ -37,6 +38,7 @@
     },
     "devDependencies": {
         "mdv": "^1.3.4",
-        "yaml": "2.4.2"
+        "yaml": "2.4.2",
+        "markdownlint-cli": "^0.27.0"

```

Comment:
i also tried updating the version to match `0.42.0` which is what I had originally, no change.  also the script uses npx, so it should grab the latest anyways. This entry is more for local development.